integration-tests/flake.nix

```diff
@@ -0,0 +1,92 @@
+{
```

Comment:
Ideally, a repo should only contain one flake.nix file. Meaning `repo === flake`.

There is some support for nested flakes (multiple in a repo) but the full support is lacking in some areas, and its not really necessary for our use cases.

The only reason why we had multiple flakes in the original [chainlink-dev repo](https://github.com/smartcontractkit/chainlink-project-nix-poc) is to represent projects as flakes as we expect projects to live in separate repos (e.g., EVM, Solana, Startknet, etc.).

We can also have a monorepo of projects, where this monorepo would also be a single flake that is a registry of N projects with M components.

Reply:
This repo already has a [Nix flake](https://github.com/smartcontractkit/chainlink/blob/develop/flake.nix) so if we keep this work in this repo (I'd prefer we move it to chainlink-evm repo), we'd need to add chainlink-dev configuration to the main flake.nix and remove this one.